nlp/optimize/tron.py

```diff
@@ -362,7 +370,7 @@ def projected_linesearch(self, x, l, u, g, d, H, alpha=1.0):
 
         # Compute the final iterate and step.
         s = projected_step(x, alpha * d, l, u)
-        x = project(x + alpha * s, l, u)
+        x = project(x + s, l, u)
```

Comment:
It's because during the call with Sylvain we realize than my TRON version was not updated and this change was maybe just a blaster to fix temporarily my issue.